src/app/technical-record/manufacturer/manufacturer.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input } from '@angular/core';
+import { ManufacturerDetails } from '@app/models/tech-record.model';
+
+@Component({
+  selector: 'vtm-manufacturer',
+  templateUrl: './manufacturer.component.html',
+  styleUrls: ['./manufacturer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ManufacturerComponent implements OnInit {
+
+  @Input() manufacturer: ManufacturerDetails;
+  address1And2 = '';
+  constructor() { }
+
+  ngOnInit() {
+    if (this.manufacturer) {
```

Comment:
this.address1And2 = ${this.manufacturer.address1} ${this.manufacturer.address2}``